services/blockchain-indexer/tests/unit/shared/dataservice/knownAccounts.test.js

```diff
@@ -0,0 +1,307 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+// Mock config
+jest.mock('../../../../config', () => ({
+	endpoints: {
+		liskStatic: 'http://mocked-static-url',
+	},
+	networks: {
+		Lisk: [
+			{ chainID: '00000000', name: 'mainnet' },
+			{ chainID: '01000000', name: 'testnet' },
+		],
+	},
+}));
+
+const chainID = '00000000';
+
+describe('reloadAccountKnowledge', () => {
+	beforeEach(() => {
+		jest.resetAllMocks();
+		jest.resetModules();
+	});
+
+	it('should log the number of entries added to the knowledge object', async () => {
+		// Mock lisk-service-framework
+		jest.mock('lisk-service-framework', () => {
+			const actual = jest.requireActual('lisk-service-framework');
+			return {
+				...actual,
+				HTTP: {
+					request: jest.fn(() => Promise.resolve({
+						data: {
+							address1: {
+								owner: 'LiskHQ',
+								description: 'Initial seed',
+							},
+							address2: {
+								owner: 'Genesis',
+								description: 'Initial seed',
+							},
+						},
+						status: 200,
+					})),
```

Comment:
Would it be more readable to move the response data to a helper constant file?
```suggestion
					request: jest.fn(() => Promise.resolve(mockedValidKnowledge)),
```